hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfspp/CMakeLists.txt

```diff
@@ -53,30 +53,14 @@ find_package(Threads)
 include(CheckCXXSourceCompiles)
 include(CheckSymbolExists)
 
-# Download and build gtest
-configure_file(CMakeLists-gtest.txt.in googletest-download/CMakeLists.txt)
```

Comment:
Why not using this approach anymore?

Reply:
In the old approach (which uses `configure_file`), CMake would first clone the gtest repo and then checkout the release tag `release-1.10.0`. I believe, something has changed in the manner through which the repo is cloned which is causing the `release-1.10.0` tag to abscond.

In the approach proposed by this PR, we do away with the clone and checkout steps, since we get the gtest source code at `release-1.10.0` directly.